nservicebus/errors/index.md

```diff
@@ -18,8 +18,9 @@ NServiceBus has exception catching and handling logic of its own which surrounds
 
 When a message fails NServiceBus [automatically retries](/nservicebus/errors/automatic-retries.md) the message. On repeated failure NServiceBus forwards that message to a designated error queue.
 
-Error queue can be configured in several ways.
+NOTE: When running with [transport transactions](/nservicebus/messaging/transactions.md) disabled NServiceBus bus will perform best-effort error message forwarding.
```

Comment:
I think we should explain what best effort means, i.e. indicate that if moving to error queue fails, the message will be lost.